lib/money/money.rb

```diff
@@ -365,34 +365,37 @@ def initialize(obj, currency = nil, options = {})
   end
 
   # Assuming using a currency using dollars:
-  # Returns the value of the money in dollars,
-  # instead of in the fractional unit cents.
+  # Returns the value of the money in dollars, instead of in the fractional unit
+  # cents.
   #
   # Synonym of #amount
   #
   # @return [BigDecimal]
   #
   # @example
-  #   Money.new(1_00, "USD").dollars   # => BigDecimal("1.00")
+  #   Money.new(1_00, "USD").dollars # => BigDecimal("1.00")
```

Comment:
I’ve included a few minor whitespace fixes to harmonize the documentation format.